### DISCUSSION TITLE
Anyone elses ACRE2 Locking up ARMA3 after the most recent teamspeak update?

### DISCUSSION THREAD

**Discussion by tobjv** [IDI-Systems/acre2#1257] [General]
As title says, others and my self that i play with have been having lock ups from teamspeak with acre loaded. it either causes arma to crash with ts or if you close ts with taskmanager you can get arma3 to recover. i dont have any logs to give atm.

**Comment by jonpas** [comment]
Known issue https://github.com/IDI-Systems/acre2/issues/1258
